src/assessments/readability/subheadingDistributionTooLongAssessment.js

```diff
@@ -23,10 +25,50 @@ class SubheadingsDistributionTooLong extends Assessment {
 		super();
 
 		let defaultConfig = {
-			// The maximum recommended value of the subheading text.
-			recommendedMaximumWordCount: 300,
-			slightlyTooMany: 300,
-			farTooMany: 350,
+			parameters: {
+				// The maximum recommended value of the subheading text.
+				recommendedMaximumWordCount: 300,
+				slightlyTooMany: 300,
+				farTooMany: 350,
+			},
+			url: "<a href='https://yoa.st/headings' target='_blank'>",
+			goodShortTextNoSubheadings: {
+				score: 9,
+				resultText: "You are not using any %1$ssubheadings%2$s, but your text is short enough and probably doesn't need them.",
+				resultTextPlural: "",
```

Comment:
Any reason for setting the empty `resultTextPlural` strings? I think they can just be removed when they're not required.

Reply:
Fixed.